script/install-protoc.sh

```diff
@@ -13,7 +13,7 @@ os=$(uname)
 arch=$(uname -m)
 
 install_dir=$HOME/.local
-$install_dir/bin/protoc --version | grep $VERSION
+protoc --version | grep $VERSION
```

Comment:
With the qualified path, it can look like the correct version is installed even when that program will not be found on the PATH.